PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -124,14 +124,19 @@ local function CreateCamera()
 			elseif cameraSubject:IsA('Model') then
 				result = cameraSubject:GetModelCFrame().p
 			elseif cameraSubject:IsA('Humanoid') then
-				local humanoidRootPart = cameraSubject.Torso
-				if humanoidRootPart and humanoidRootPart:IsA('BasePart') then
-					local subjectCFrame = GetRenderCFrame(humanoidRootPart)
-					result = subjectCFrame.p +
-						subjectCFrame:vectorToWorldSpace(HEAD_OFFSET + cameraSubject.CameraOffset)
+				if cameraSubject:GetState() == Enum.HumanoidStateType.Dead and HasVRAPI and UserInputService.VREnabled then
+					result = this.lastSubjectPosition
```

Comment:
I prefer to use `self` instead of `this` when accessing member variables in a member function